client/src/components/Tickets/TicketModal.js

```diff
@@ -34,22 +36,57 @@ const formSchema = Yup.object().shape({
   reply: Yup.string().required('Required')
 });
 const Attachments = ({ attachments, ticketId }) => {
+  const infiniteScrollCallback = useCallback(() => {});
+  const noDataText = 'No attachments to display.';
+  const json = attachments.map(function attachmentAcessor(x) {
+    return {
+      attachment_id: x[0],
+      attachment_name: x[1]
+    };
+  });
+
+  const columns = [
+    {
+      Header: 'Attachment Files',
+      accessor: 'attachment_name',
+      className: 'attachment-title',
+      Cell: el => (
+        <span
+          title={el.value}
+          id={`attachment${el.row.index}`}
+          className="attachment__name"
+        >
+          {el.value}
+        </span>
+      )
+    },
+    {
+      Header: '',
+      className: 'link',
+      accessor: 'attachment_id',
+      Cell: el => (
+        <a
+          href={`/api/tickets/${ticketId}/attachment/${el.value}`}
+          className="link"
+          target="_blank"
+          rel="noreferrer noopener"
+          key={el.value}
+        >
+          Download
+        </a>
+      )
+    }
+  ];
   return (
     <div>
-      Attachments:
-      <ul>
-        {attachments.map(attachmentName => (
-          <a
-            href={`/api/tickets/${ticketId}/attachment/${attachmentName[0]}`}
-            className="link"
-            target="_blank"
-            rel="noreferrer noopener"
-            key={attachmentName[0]}
-          >
-            <li> {attachmentName[1]} </li>
-          </a>
-        ))}
-      </ul>
+      <InfiniteScrollTable
```

Comment:
**Question.** Do we expect to have a list of attachments that is long enough to benefit from infinite scroll?

<details>
<summary>Thoughts</summary>

If `<InfiniteScrollTable>` were **not** used, we could likely borrow its styles and fixed header [as Allocations does](https://github.com/TACC/Core-Portal/blob/0efcc92/client/src/components/Allocations/AllocationsTables.js#L110-L114).

</details>

Reply:
[I asked Design.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1636492013059700)

Reply:
**Yes**, why not — [Slack thread](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1636601801060700?thread_ts=1636492013.059700&cid=CQUA4D5KJ), 2021-11-11

Reply:
[UI - Behavior / UX - Tables](https://confluence.tacc.utexas.edu/x/D4BCCQ) is now up to date.